### ISSUE TITLE
Jobbörse von www.workeer.de einbinden

### ISSUE THREAD

**Issue by matsair** [digitalfabrik/app-android#88]
Nur möglich wenn der User eingeloggt ist. Müsste man mal ausarbeiten.

**Comment by Dalanie** [comment]
hat workeer eine api?

**Comment by sven15** [comment]
FYI: Michael Mittag hatte sich im Umfeld Jobbörsen schlau gemacht und geschaut, welche was taugen und welche nicht. Ich treffe ihn am Freitag Mittag und werde das Thema noch einmal anschneiden.

**Comment by dkehne** [comment]
Workeer ist allein von den Personen dahinter, der Vision und der Bekanntheit eigentlich idealer Partner. Wenn wir so Geschichten wie "Monster" oder so anfangen, dann können wir die API Verzahnung zu einem eigenen Projekt machen. Aktuell haben die beiden Workeer-Jungs noch keine API glaube ich. @matsair kannst du die mal anschreiben und nachhaken? Wäre eh erst ein Release 2.0 Punkt

**Comment by matsair** [comment]
@dkehne ich schreibe die mal an. hatte einer von uns schonmal kontakt mit denen?

**Comment by maegnes** [comment]
Wir haben für ein anderes Projekt diesbezüglich bereits bei workeer angefragt - allerdings nie eine Antwort erhalten. Die sind ziemlich Land unter.

Hat von euch jemand inzwischen eine Info bekommen?

**Comment by dkehne** [comment]
Aktuell noch nicht, aber das Thema greifen wir ab Mitte Februar wieder auf, wenn wir die gesamten Arbeitsmarkt-Projekte, wie z.B. auch Arrival Talents, mal überblickt haben

**Comment by maegnes** [comment]
Wir hatten mal einen direkten Kontakt zu workeer - wir werden es auch hierüber nochmal versuchen. Sollte ich etwas erfahren lasse ich es euch natürlich wissen.

**Comment by maegnes** [comment]
Laut Aussage von workeer ist aktuell kein API vorhanden. Allerdings wird die Plattform in den nächsten Monaten komplett modernisiert. Dann lässt sich ein API auch leichter integrieren als momentan. Sie sind also über die Notwendigkeit informiert und haben es auf dem Schirm.

**Comment by dkehne** [comment]
Stark. Danke fürs Nachhaken! :) bleibst du da am ball?

**Comment by maegnes** [comment]
Klar. Ich habe zwar mit dem Projekt hier nichts zu tun - aber natürlich halte ich euch auf dem Laufenden! ;)
